e2e/test/scenarios/sharing/public-sharing-embed-button-behavior.cy.spec.js

```diff
@@ -225,11 +230,19 @@ describe("embed modal display", () => {
         cy.findByText("Static embedding").should("be.visible");
         cy.findByText("Interactive embedding").should("be.visible");
 
-        cy.findByRole("link", { name: /Interactive embedding/ }).should(
+        cy.findByRole("link", { name: "Interactive embedding" }).should(
```

Comment:
modify the assertion for [this case](https://www.notion.so/metabase/Streamline-embed-flow-when-embedding-features-are-off-13869354c90180b4bc02ca29ebf61ad2?pvs=4#14c69354c90180748578f93e02f89579): unavailable.